src/test/errors-analysis.test.ts

```diff
@@ -1696,11 +1696,6 @@ test(`we don't produce a duplicate not found error when there's multiple deps in
       'child/package.json'
     )}"
             "./child:error1",
-             ~~~~~~~
-❌ package.json:9:10 package.json file missing: "${rig.resolve(
```

Comment:
We now only report the first package-level error for a package, and only the first error for a given script. I was seeing some duplicate failure events.

My thinking is that when there are multiple issues with a package or script, that the first one is probably all that you need, and the rest just add noise. e.g. if you accidentally deleted or moved a package.json file, only the first message telling you that it can't be found is really helpful.